source/localizable/tutorial/autocomplete-component.md

```diff
@@ -250,3 +254,6 @@ export default function() {
 }
 ```
 
+After updating your mirage config, you should see passing tests, as well as a simple filter on your home screen, that will update the rental list as you type:
```

Comment:
We generally speak in a "we" or plural form, but this sentence shifts the other way.  Let's update so it reads more like:

> After updating our Mirage config, we should see passing tests